spec/unit/provider/group/windows_adsi_spec.rb

```diff
@@ -35,11 +35,13 @@
     let(:user1) { stub(:account => 'user1', :domain => '.', :to_s => 'user1sid') }
     let(:user2) { stub(:account => 'user2', :domain => '.', :to_s => 'user2sid') }
     let(:user3) { stub(:account => 'user3', :domain => '.', :to_s => 'user3sid') }
+    let(:invalid_user) { SecureRandom.uuid }
```

Comment:
ACLs will continue to contain deleted accounts, but I can't seem to get it to show me a user that doesn't exist in a group on Windows. In short, I'm not sure how to get a real-life scenario where puppet resource group would spit out a user that is invalid.